pyloopkit/insulin_math.py

```diff
@@ -351,15 +361,17 @@ def annotated(
     output_end_dates = []
```

Comment:
@novalegra Just hit an error 4 lines above this where the early return contains only 5 empty lists, instead of 6

Reply:
Sorry about that, will fix and look for any other instances where this could happen

Reply:
Just pushed the fix to the branch, thankfully I couldn’t find any other instances of this happening